### PR TITLE
Describe new monitoring metrics for salt queue

### PR DESCRIPTION
**Pull request by mcalmer**
# Description

New metrics for salt queue will be added to SUMA. This PR describe them.

# Target branches

* Which documentation version does this PR apply to?
* Does this content need to be backported? If this is the case, create an issue for tracking it and add the link to this PR.
* Whenever possible, cross-reference each backport PR here, so that all backports can be easily accessed from the description.

- [x] master - https://github.com/uyuni-project/uyuni/pull/8050
- [x] manager-4.3 - https://github.com/SUSE/spacewalk/pull/23225

# Links
- https://github.com/uyuni-project/uyuni/pull/8050
- https://github.com/SUSE/spacewalk/pull/23225

### CODE REVIEW

File: modules/administration/pages/monitoring.adoc [hunk 1]
```diff
@@ -407,6 +407,32 @@ The following relevant metrics are collected on the {productname} Server.
...
+|===
+|Metric |Type | Description
+
+|salt_queue_thread_pool_threads
+|counter
+|Number of salt queues ever created
+
+|salt_queue_thread_pool_threads_active
+|gauge
+|Number of currently active salt queues
```
Comment (keichwa) [inline modules/administration/pages/monitoring.adoc:421]:
```suggestion
|Number of currently active {salt} queues
```

**Review by keichwa — APPROVED** [review]
cosmetic suggestions.  we have a `{salt}` macro, let's use it.

**Comment by mcalmer** [comment]
I have changed a lot. So another review would be good

**Review by keichwa — APPROVED** [review]
If I click "Files changed" I only see the already approved changes.  The 2 force-pushed actions don't seem to count as changes.

**Comment by mcalmer** [comment]
Code is merged

**Pull request merged**